frontend/src/pages/_app.jsx

```diff
@@ -73,9 +71,7 @@ export default class extends App {
             <div className="app-wrapper">
               <Header />
               <div className="page">
-                <Container>
-                  <Component {...pageProps} />
-                </Container>
```

Comment:
I see why you want to get rid of `Container`, but let's move it to `frontend/src/components/utils/Content.tsx`, so we don't need to change all the pages with an extra irrelevant `<Container>` wrapper.

Reply:
Done, with a mention: In /account/[id] the `<AccountDetails>` component had to be wrapped in a `<Container>` because inside <AccountDetails> there is no `<Content>` and i would add it but there are a some props that should be passed and idk yet the values for it. If you think that `<AccountDetails>` should be updated instead of that wrapper please provide more details 💯

Reply:
`<AccountDetails>` is currently wrapped in `<Content>`, so if you move `<Container>` into `<Content>`, it will be fine. Did you mean `<ContractDetails>`? Just wrap it with `<Container>` in the `src/pages/accounts/[id].jsx`.

Reply:
My bad, yes, i was talking about `<ContractDetails>` and i wrapped it in a `<Container>`

Reply:
https://github.com/near/near-explorer/pull/335/commits/c07dfa6424e6c747303af74e0d8bcac3219b2524